test/upgrade/service_postupgrade_test.go

```diff
@@ -62,6 +72,30 @@ func TestBYORevisionPostUpgrade(t *testing.T) {
 	}
 }
 
+func configHasGeneration(t *testing.T, clients *test.Clients, serviceName string, generation int64) bool {
+	configObj, err := clients.ServingAlphaClient.Configs.Get(serviceName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatalf("Failed to get Configuration: %v", err)
```

Comment:
Missed this before, but in a helper function you shouldn't fail the test. I would instead recommend returning `bool, error`.  This also means you don't need to pass in the testing client into the function.